site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Storage and cookies"
+seoTitle: "Chrome extensions storage and cookies"
+date: 2023-08-01
+description: Overview of how web storage APIs and cookies work in extensions.
+---
+
+Extensions can store cookies and access web storage APIs similarly to a normal website. However, in
+some cases these behave differently in extensions.
+
+See [`chrome.storage`][chrome-storage-api] for information on the extension API.
+
+## Storage {: #storage }
+
+It is often desirable to use web platform storage APIs in extensions. This section explores the
+behavior of these APIs in an extension context, which can sometimes differ with how they behave on the
+web.
+
+### Persistence {: #storage-persistence }
+
+Extension storage is not cleared when a user ["clears browsing data"][clear-browsing-data].
+This applies to any data stored using web storage APIs (such as
+[Local Storage][local-storage] and [IndexedDB][indexeddb]).
+
+By default, extensions are subject to the normal quota restrictions on storage, which can be checked
+by calling `navigator.storage.estimate()`. Storage can also be evicted under heavy memory
+pressure, although this is rare. To avoid this:
+
+- Request the `"unlimitedStorage"` permission, which affects both extension and web storage APIs and
+exempts extensions from both quota restrictions and eviction.
+- Call [`navigator.storage.persist()`][storage-persist] for protection against eviction.
+
+Extension storage is shared across the extension's origin including the extension service worker,
+any extension pages (including popups and the side panel), and offscreen documents. In content
+scripts, calling web storage APIs accesses data from the host page the content script is injected on
+and not the extension.
+
+### Access in service workers {: #storage-in-service-workers }
+
+Some web storage APIs like [IndexedDB][indexeddb] are accessible in service workers. However,
+[Local Storage][local-storage] and [Session Storage][session-storage] are not.
+
+If you have a specific use case, use an [offscreen document][offscreen] to access these.
+
+### Partitioning {: #storage-partitioning }
+
+Partitioning is where keys are introduced for stored data to limit where it can be accessed.
+
+When enabled, [storage partitioning][storage-partitioning] introduces changes to how partitioning
```

Comment:
This is another example, of why I made a proposal for how to deal with items behind a flag. We shouldn't be talking about this in primary documentation yet.